agents/agent_engine/tutorial_get_started_with_agent_engine_terraform_deployment.ipynb

```
@@ -0,0 +1,1427 @@
+      },
+      "outputs": [],
+      "source": [
+        "# Init file\n",
+        "with open(\"./agent-engine-terraform/01-basic-agent/agent/__init__.py\", \"w\"):\n",
+        "    pass\n",
+        "\n",
+        "# Create a custom agent following the Vertex AI Agent Engine pattern\n",
+        "simple_agent_code = \"\"\"\n",
+        "from typing import Dict\n",
+        "\n",
+        "class SimpleAgent:\n",
```

Ruff (0.14.3)

`open()` should be replaced by `Path.open()`

(PTH123)